src/containers/LoginContainer/index.js

```diff
@@ -1,106 +1,32 @@
-      return false;
-    }
-
-    return true;
-  }
-
-  const submit = async () => {
-    if (!validateForm()) { return }
-
-    const authenticated = await login({ username, password });
-    if (authenticated) {
-      const from = location.state?.from?.pathname || '/';
-      navigate(from, { replace: true });
-    } else {
-      setErrorMessage('login.username_or_password_incorrect');
-    }
-  }
-
-  return (
-    <CardGrid>
-      <Row>
-        <Col xs={12} sm={8} smOffset={2} md={6} mdOffset={3}>
-          <Card className="login">
-            <div className="logo" />
-            <CardBody>
-              <Form horizontal>
-                <h1 className="text-center">{T.translate('login.header')}</h1>
-                <Row>
-                  <Col xs={8} xsOffset={2}>
-                    <FormGroup controlId='username' bsSize='large'>
-                      <ControlLabel>
-                        {T.translate('login.username')}
-                      </ControlLabel>
-                      <FormControl
-                        name='username'
-                        autoFocus
-                        placeholder={T.translate('login.username')}
-                        onChange={({ target: { value } }) => setUsername(value)}
-                      />
-                    </FormGroup>
-                  </Col>
-                </Row>
-                <Row>
-                  <Col xs={8} xsOffset={2}>
-                    <FormGroup controlId='password' bsSize='large'>
-                      <ControlLabel>
-                        {T.translate('login.password')}
-                      </ControlLabel>
-                      <FormControl
-                        name='password'
-                        type='password'
-                        placeholder={T.translate('login.password')}
-                        onChange={({ target: { value } }) => setPassword(value)}
-                      />
-                    </FormGroup>
-                  </Col>
-                </Row>
-                { errorMessage && <Alert type="error" message={errorMessage} /> }
-                <Row>
-                  <Col xs={8} xsOffset={2}>
-                    <FormGroup>
-                      <Button className="login" bsStyle='primary' bsSize='large' onClick={submit}>
-                        {T.translate('login.login')}
-                      </Button>
-                    </FormGroup>
-                  </Col>
-                </Row>
-              </Form>
-            </CardBody>
-          </Card>
-        </Col>
-      </Row>
-    </CardGrid>
-  )
-}
+const LoginContainer = () => (
+  <Card isCompact isFlat className={css(sizing.w_50, spacing.mxAuto, spacing.pXl)}>
+    <CardTitle>
+      <Brand src={logo} alt="LISA" className={css(sizing.w_100, alignment.textAlignCenter)} />
+    </CardTitle>
+    <CardBody>
+      <Flex direction={{ default: 'column' }}>
+        <FlexItem>
+          <ForemanAuth />
```

Comment:
what about making this also optional so a admin could disable regular foreman auth with a setting?